src/components/navigation/right-nav.tsx

```diff
@@ -65,28 +65,26 @@ export class RightNavComponent extends BaseComponent<IProps, IState> {
 
   public render() {
     const {activeRightNavTab, rightNavExpanded} = this.stores.ui;
-    const teacherTabs = [kClassWorkTab, kClassLogsTab];
-    const studentTabs = [kMyWorkTab].concat(teacherTabs);
-    const tabs = this.props.isGhostUser ? teacherTabs : studentTabs;
+    const tabSpecs = this.props.tabs
+                      .filter(tabSpec => !(this.props.isGhostUser && tabSpec.hideGhostUser));
```

Comment:
If `tabSpecs` is empty, should we even render `<div className="right-nav">`?  If we have a case where we are not using the right tabs at all, we will still see the ~16 pixel edge of the right nav div sticking out on the right edge of the screen.  However, if we never plan to have a case where there are 0 tabs, then this point is moot.

Reply:
I thought about that briefly but didn't pursue it; I think that's right, though. I'll make that change.

Reply:
At one point I recall the DF folks requesting no tabs on the right side, but that was several weeks ago.  Not sure what the current plan is.

Reply:
Regardless of what the current plan is, no tabs should be a supported configuration.

Reply:
👍